src/components/BlogList.jsx

```diff
@@ -14,6 +14,7 @@ function BlogList() {
   let lastDisplayOfPage = firstDisplayOfPage + rowsPerPage;
 
   const updateRowsPerPage = (e) => {
+    console.log('code to prevent identical for push purposes');
```

Comment:
Console.log statements are good for testing, but not good practice for production, and should be removed.

Reply:
Hi there Aman,

Thank you for the feedback and suggestions! I went ahead and reverted the main branch to its original form. I also went ahead and followed the prompt for the page reset on the page option selection to go back to "1".

I will resubmit my application and wait for the tests to run!